tests/cephadm/snmp_gateway.py

```diff
@@ -0,0 +1,37 @@
+"""Module to deploy SNMP-Gateway service and individual daemon(s)."""
```

Comment:
we need to move this under lib (ceph/cehadmin).

Reply:
Moved to ceph/cephadm and file exists for CLI deployment further